scripts/export-framework-applicant-details.py

```diff
@@ -37,7 +37,7 @@
     client = DataAPIClient(get_api_endpoint_from_stage(STAGE), API_TOKEN)
     now = datetime.datetime.now()
 
-    filename = OUTPUT_DIR + os.sep if OUTPUT_DIR else ""
+    filename = (OUTPUT_DIR if OUTPUT_DIR else ".") + os.sep
```

Comment:
I think I'd still rather be explicit about the target directory, and otherwise just see everything concatenated on stdout, myself :) Otherwise one can overwrite things one did not expect to..